docs/proposals/scheduling/20220629-fine-grained-device-scheduling.md

```diff
@@ -303,61 +303,57 @@ Implements a new component called  `Device Reporter` in koordlet to create or up
 type DeviceType string
 
 const (
-    GPU  DeviceType = "gpu"
-    FPGA DeviceType = "fpga"
-    RDMA DeviceType = "rdma"
+	GPU  DeviceType = "gpu"
+	FPGA DeviceType = "fpga"
+	RDMA DeviceType = "rdma"
 )
 
-type DeviceList struct {
-    metav1.TypeMeta `json:",inline"`
-    metav1.ListMeta `json:"metadata,omitempty"`
-
-    Items []Device `json:"items"`   
+type DeviceSpec struct {
+	Devices []DeviceInfo `json:"devices"`
 }
 
-type DevicesSpec struct {
-    Items  []Device `json:"items"`
+type DeviceInfo struct {
+	// UUID represents the UUID of device
+	UUID string `json:"id,omitempty"`
+	// Minor represents the Minor number of Device, starting from 0
+	Minor int32 `json:"minor,omitempty"`
+	// Type represents the type of device
+	Type DeviceType `json:"deviceType,omitempty"`
+	// Health indicates whether the device is normal
+	Health bool `json:"health,omitempty"`
+	// Resources represents the total capacity of various resources of the device
+	Resources map[string]resource.Quantity `json:"resource,omitempty"`
 }
 
-type Device struct {
-    metav1.TypeMeta   `json:",inline"`
-    metav1.ObjectMeta `json:"metadata,omitempty"`
-
-    Spec   DeviceSpec `json:"spec,omitempty"`
-    Status DevicesStatus `json:"status,omitempty"`
+type DeviceStatus struct {
+	Allocations []DeviceAllocation `json:"allocations"`
```

Comment:
Why register the allocation status here?

Reply:
I didn't bring Allocations into DeviceStatus in this PR. For more detail, you might wanna check with @buptcozy.